src/Aspire.Hosting.Azure.EventHubs/AzureEventHubsExtensions.cs

```diff
@@ -148,8 +148,7 @@ public static IResourceBuilder<AzureEventHubsResource> RunAsEmulator(this IResou
             return builder;
         }
 
-        var configHostFile = Path.Combine(Path.GetTempPath(), "AspireEventHubsEmulator", Path.GetRandomFileName() + ".json");
-        Directory.CreateDirectory(Path.GetDirectoryName(configHostFile)!);
+        var configHostFile = Path.Combine(Directory.CreateTempSubdirectory("AspireEventHubsEmulator").FullName, "Config.json");
```

Comment:
@eerhardt Longer term we should have a service for doing this that gets cleaned up on exit. @karolz-ms maybe something DCP can provide since it controls the clean up.

Reply:
Definitely. For example, I can imagine us having the capability for Containers where you can say "here is a blob of JSON that I need to appear as file inside the container, under this path". Sort of like Kubernetes ConfigMaps.

Reply:
Opened https://github.com/dotnet/aspire/issues/7029 to track.